### PR TITLE
Merge with ember-class-based-modifier

### LINKED ISSUE #9: Glimmer component unification
Hi 👋,

would it make sense to unify the native `Modifer` API with Glimmer component API?

https://github.com/glimmerjs/glimmer.js/blob/master/packages/%40glimmer/component/addon/-private/component.ts#L147-L158

This would need to change the order of `constructor` arguments and using `args` property.

```javascript
class Modifier {
  // before
  constructor(attrs = {}, _owner) {
    setOwner(this, _owner);
    setProperties(this, attrs);
  }

  // after
  constructor(owner, args) {
    this.args = args;
    setOwner(this, owner);
  }

  // ...
}
```

Thanks

### PR DESCRIPTION
**Pull request by chancancode**
This is a result of squashing everything [after the v0.5.0 tag up to and including the v0.9.0 tag](https://github.com/sukima/ember-oo-modifiers/compare/v0.5.0..v0.9.0), rebased on top of the current master, which is the [v0.5.2](https://github.com/sukima/ember-oo-modifiers/tree/v0.5.2) tag.

The original commits are at the [archive/ember-class-based-modifier-fork branch](https://github.com/sukima/ember-oo-modifiers/tree/archive/ember-class-based-modifier-fork) as a permanent archive, but the commits themselves are not very interesting. It's mainly there so that the [v0.8.0](https://github.com/sukima/ember-oo-modifiers/tree/v0.8.0) and [v0.9.0](https://github.com/sukima/ember-oo-modifiers/tree/v0.9.0) tags have something to point to.

* [x] I have added @sukima as an NPM owner of `ember-class-based-modifier`
* [x] @sukima: If you can add me as an NPM owner of `ember-oo-modifiers`, just in case we need to cut more maintenance releases?
  * [x] I have created a [maintenance branch](https://github.com/sukima/ember-oo-modifiers/tree/maintenance/ember-oo-modifiers) for that purpose
* [x] @sukima: Can you rename this repo to `ember-class-based-modifier` in the repo Settings? I believe it could be done either before or after the merge.
* [x] @sukima: Can you enable Travis CI in the repo Settings? Maybe Travis would be less confused if you do this after the rename?
* [x] @sukima: Can you update the repo's description and tags? (The bar above the file browser but below the "Code", "Issues", ... tabs.)

Thank you very much for the fast turnaround!!

Closes #9 and #14!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ember-class-based-modifier
 ==============================================================================
 
-This is a fork of [ember-oo-modifiers](https://github.com/sukima/ember-oo-modifiers) with some modifications to the API.
+[![Build Status](https://travis-ci.org/sukima/ember-class-based-modifier.svg?branch=master)](https://travis-ci.org/chancancode/ember-class-based-modifier)
+
+> This is the next iteration of [ember-oo-modifiers](https://github.com/sukima/ember-class-based-modifier/tree/maintenance/ember-oo-modifiers) with some breaking changes to the API. If you are looking for the previous documentation, select the versoin you are using in the tags dropdown at the top of the page. For a list of API differences, see [here](#api-differences-from-ember-oo-modifiers)
```
Comment (sukima) [inline README.md]: `versoin` => `version`

Comment (chancancode) [reply README.md]: fixed!

**Review by sukima — APPROVED** [review]

**Comment by sukima** [comment]
@chancancode Done, Should be ready to merge. I think you have access to all that

**Comment by chancancode** [comment]
@sukima looks good to me! CI is green to so I think we can merge this any time!

```
$ npm owner ls ember-oo-modifiers
sukima <suki@tritarget.org>
```

I don't have access to ember-oo-modifiers yet, but I think this takes a long time to propagate on NPM, and it's not urgent anyway. I assume you may also have the opposite issue?

**Pull request merged**